docs/API/panel/table/index.md

```diff
@@ -282,7 +282,7 @@ PARAMETERS:
 * **value** (`boolean`)
    - default value: `true`
 
-
+?? default is missing or false ??
```

Comment:
huh?

Reply:
That comes from the schemas, I just scanned for that sentence but can't find the source immediately, cc/ @K-Phoen, do you know?

Reply:
This comment comes from https://github.com/grafana/kind-registry/blob/f4b454b319b314e2965ca0d1a102677ab79f5f93/grafana/v11.0.x/common/table.cue#L93 :/

Reply:
Thanks, I'll leave it as such.